app/main.py

```diff
@@ -1,34 +1,68 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.col = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self, start: tuple, end: tuple,
+                 is_drowned: bool = False) -> None:
 
-    def get_deck(self, row, column):
-        # Find the corresponding deck in the list
-        pass
+        self.is_drowned = is_drowned
+        self.decks = []
+        if start[0] == end[0]:
+            for col in range(start[1], end[1] + 1):
+                self.decks.append(Deck(start[0], col))
+        elif start[1] == end[1]:
+            for row in range(start[0], end[0] + 1):
+                self.decks.append(Deck(row, start[1]))
```

Comment:
You can use loop into loop instead of if statements
```suggestion
        for col in range(start[1], end[1] + 1):
            for row in range(start[0], end[0] + 1):
                self.decks.append(Deck(row, col))
```